src/Akka.Streams.Kafka/Stages/Consumers/Abstract/SubSourceLogic.cs

```diff
@@ -264,41 +375,34 @@ private void OffsetsFromExternalResponseCallback((IImmutableSet<TopicPartition>,
                 offsets.Where(o => !_partitionsToRevoke.Contains(o.TopicPartition)).ToImmutableHashSet());
         }
 
-        private void SeekAndEmitSubSources(IImmutableSet<TopicPartition> formerlyUnknown, IImmutableSet<TopicPartitionOffset> offsets)
+        private void SeekAndEmitSubSources(IImmutableSet<TopicPartition> formerlyUnknown,
+            IImmutableSet<TopicPartitionOffset> offsets)
         {
-            ConsumerActor.Ask(new KafkaConsumerActorMetadata.Internal.Seek(offsets), TimeSpan.FromSeconds(10))
-                .ContinueWith(t =>
+            _ = AskToSeekOffsets();
+            return;
+
+            async Task AskToSeekOffsets()
+            {
+                try
                 {
-                    if (t.IsCanceled)
```

Comment:
Simplified the error-handling here a great deal but just using a local `async Task` function and a `try...catch` block